minimal-notebook/test/test_nbconvert.py

```diff
@@ -19,17 +21,19 @@
         ("notebook_svg", "html"),
     ],
 )
-def test_nbconvert(container, test_file, output_format):
+def test_nbconvert(
+    container: TrackedContainer, test_file: str, output_format: str
+) -> None:
     """Check if nbconvert is able to convert a notebook file"""
-    host_data_dir = os.path.join(THIS_DIR, "data")
+    host_data_dir = THIS_DIR / "data"
     cont_data_dir = "/home/jovyan/data"
     output_dir = "/tmp"
     LOGGER.info(
         f"Test that the example notebook {test_file} can be converted to {output_format} ..."
     )
     command = f"jupyter nbconvert {cont_data_dir}/{test_file}.ipynb --output-dir {output_dir} --to {output_format}"
     c = container.run(
-        volumes={host_data_dir: {"bind": cont_data_dir, "mode": "ro"}},
+        volumes={str(host_data_dir): {"bind": cont_data_dir, "mode": "ro"}},
```

Comment:
The same situation I commented on above.

Reply:
Nevermind, let's go with this, I see in other situations you have used functions on the returned Path object even though you don't do it here.